modules/custom/az_accordion/templates/az-accordion.html.twig

```diff
@@ -15,25 +15,19 @@
  */
 #}
 
-{% set attributes = attributes.addClass('h-100') %}
-
-<div {{ attributes }}>
-  <div class="card">
-    <div class="card-header">
-      <h3 class="my-0">
-        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#{{ accordion_item_id }}" aria-expanded="{{ aria_expanded }}" aria-controls="{{ accordion_item_id }}">
-          {% if title %}
-            {{ title }}
-          {% endif %}
-        </button>
-      </h3>
-    </div>
-    <div id="{{ accordion_item_id }}" class="{{ collapsed }}" aria-labelledby="{{ accordion_item_id }}" data-parent="#{{ accordion_item_id }}">
-      <div class="card-body bg-white">
-        {% if body %}
-        <div class="accordion-text">{{ body }}</div>
-        {% endif %}
-      </div>
-    </div>
+<div class="accordion-header">
+  <h3 class="my-0">
+    <button class="accordion-button collapsed" type="button" data-toggle="collapse" data-target="#{{ accordion_item_id }}" aria-expanded="{{ aria_expanded }}" aria-controls="{{ accordion_item_id }}">
+      {% if title %}
+        {{ title }}
+      {% endif %}
+    </button>
+  </h3>
+</div>
+<div id="{{ accordion_item_id }}" class="accordion-collapse {{ collapsed }}" aria-labelledby="{{ accordion_item_id }}" data-parent="#{{ accordion_item_id }}">
+  <div class="card-body bg-white">
```

Comment:
Investigate use of card-body here

Reply:
AZBS 5 (no .card-body with accordions): https://review.digital.arizona.edu/arizona-bootstrap/main/docs/5.0/components/accordion/ 
AZBS 2 (does use .card and .card-body): https://digital.arizona.edu/arizona-bootstrap/docs/2.0/components/collapse/